ndarray/ndarray-core/src/commonMain/kotlin/io/kinference/ndarray/extensions/argMinMax/PrimitiveArgMinMax.kt

```diff
@@ -0,0 +1,128 @@
+@file:GeneratePrimitives(DataType.NUMBER)
+
+package io.kinference.ndarray.extensions.argMinMax
+
+import io.kinference.ndarray.arrays.*
+import io.kinference.ndarray.arrays.tiled.PrimitiveTiledArray
+import io.kinference.primitives.annotations.GenerateNameFromPrimitives
+import io.kinference.primitives.annotations.GeneratePrimitives
+import io.kinference.primitives.types.DataType
+import io.kinference.primitives.types.PrimitiveType
+
+@GenerateNameFromPrimitives
+internal fun argMinMaxPrimitive(input: PrimitiveNDArray, axis: Int, keepDims: Boolean, selectLastIndex: Boolean, isMin: Boolean = false): IntNDArray {
```

Comment:
It'll be more readable if you choose suitable comparator inside PrimitiveNDArray argmin/argmax functions and pass it here instead of isMin flag